t/management-policies-endpoint-custom.t

```diff
@@ -0,0 +1,55 @@
+use lib 't';
+use Test::APIcast::Blackbox 'no_plan';
+
+BEGIN {
+    $ENV{TEST_NGINX_MANAGEMENT_SERVER_NAME} = 'management';
+}
+
+env_to_apicast(
+    'APICAST_CONFIGURATION_LOADER' => 'test',
+    'APICAST_POLICY_LOAD_PATH' => "$ENV{PWD}/t/fixtures/policies_endpoint_test/policies"
+);
+
+# Converts what's in the 'expected_json' block and the body to JSON and
+# compares them. Raises and error when they do not match.
+add_response_body_check(sub {
+    my ($block, $body) = @_;
+
+    use JSON;
+    use Data::Compare;
+
+    my $h1 = JSON->new->utf8->decode($body);
+    my $h2 = JSON->new->utf8->decode($block->expected_json);
+    my $c = Data::Compare->new($h1, $h2);
+    if (!$c->Cmp) {
+        bail_out "JSON returned does not match the expected one.";
+    }
+
+});
+
+repeat_each(1);
```

Comment:
Any reason to repeat just once instead of the default 2 ?

Reply:
I always forget about this and copy it from the wrong place. I'll just delete this as Test::APIcast uses repeat_each(2) by default when not set.

Reply:
Perils of copy&pasting ☠️